client/src/app/site/tags/components/tag-list/tag-list.component.ts

```diff
@@ -5,7 +5,7 @@ import { MatDialog, MatDialogRef } from '@angular/material/dialog';
 import { TranslateService } from '@ngx-translate/core';
 import { PblColumnDefinition } from '@pebula/ngrid';
 
-import { SimplifiedModelRequest } from 'app/core/core-services/model-request-builder.service';
+import { SimplifiedModelRequest, DEFAULT_FIELDSET } from 'app/core/core-services/model-request-builder.service';
```

Comment:
> DEFAULT_FIELDSET

can this be removed? They are usually in the repositories

Reply:
yes, right.

Reply:
welp matters little. Cleanup will eventually come :D